content/02.introduction.md

```diff
@@ -13,12 +13,12 @@ Gender bias appears to also influence funding decisions: an examination of scori
 Challenges extend beyond gender: an analysis of awards at the NIH found that proposals by Asian, black or African-American applicants were less likely to be funded than those by white applicants [@doi:10.1126/science.1196783].
 There are also potential interaction effects between gender and race or ethnicity that may particularly affect women of color's efforts to gain NIH funding [@doi:10.1097/ACM.0000000000001278].
 
-We sought to understand the extent to which honors and high-profile speaking invitations were distributed equitably among gender, race/ethnicity, and nationality groups by an international society and its associated meetings.
+We sought to understand the extent to which honors and high-profile speaking invitations were distributed equitably among gender, race/ethnicity, and name etymology groups by an international society and its associated meetings.
 As computational biologists, we focused on the [International Society for Computational Biology](https://www.iscb.org/) (ISCB), its honorary Fellows as well as its affiliated international meetings: [Intelligent Systems for Molecular Biology](https://en.wikipedia.org/wiki/Intelligent_Systems_for_Molecular_Biology) (ISMB), [Research in Computational Molecular Biology](https://en.wikipedia.org/wiki/Research_in_Computational_Molecular_Biology) (RECOMB), and [Pacific Symposium on Biocomputing](https://psb.stanford.edu/) (PSB).
 
-We used multiple methods to predict the gender, race/ethnicity, and nationality of honorees.
+We used multiple methods to predict the gender, race/ethnicity, and name etymology of honorees.
 Existing methods were relatively US-centric because most of the data was derived in whole or in part from the US Census.
-We scraped more than 700,000 entries from English-language Wikipedia that contained nationality information to complement these existing methods and built multiple machine learning classifiers to predict nationality.
+We scraped more than 700,000 entries from English-language Wikipedia that contained name etymology information to complement these existing methods and built multiple machine learning classifiers to predict name etymology.
```

Comment:
Same. It's misleading to say that the Wikipedia entries contained name etymology information. I'd stick with nationality in the first part of the sentence, and then say name etymology in the second.

Reply:
With that in mind, I think I honestly prefer "name origins" over "name etymology", since the Wiki data will be loosely based on country of origin but not really on linguistics.